src/control/server/storage/bdev/backend.go

```diff
@@ -94,46 +91,6 @@ func (w *spdkWrapper) suppressOutput() (restore func(), err error) {
 	return
 }
 
-func (w *spdkWrapper) init(log logging.Logger, spdkOpts spdk.EnvOptions) (err error) {
```

Comment:
init was only ever called as part of Scan() , makes sense to me to keep things self-contained

Reply:
Hmm, but now you've got two places to maintain the logic of suppressing output, calling InitSPDKEnv(), and discovering devices. If new methods are added, those will also have to copy/paste this logic. It seems like it would be cleaner and more maintainable to pull this common code into a shared function.

Reply:
okay will reinstate